ChronoGrapher/GrapherRecordingService.h

```diff
@@ -91,10 +91,19 @@ class GrapherRecordingService: public tl::provider <GrapherRecordingService>
 
     void deserializedWithCereal(char *buffer, size_t size, StoryChunk &story_chunk)
     {
-        std::stringstream ss;
-        ss.write(buffer, size);
-        cereal::BinaryInputArchive iarchive(ss);
-        iarchive(story_chunk);
+        try
+        {
+            std::stringstream ss;
+            ss.write(buffer, size);
+            cereal::BinaryInputArchive iarchive(ss);
+            iarchive(story_chunk);
+        }
```

Comment:
TODO:  create a ticket to handle the case of failed deserialization as failed transmission of the StoryChunk . if deserialize fails we should not proceed to pass the invalid story_chunk to the grapher ingestion queue but should instead return the error to the recording service client

Reply:
> TODO: create a ticket to handle the case of failed deserialization as failed transmission of the StoryChunk . if deserialize fails we should not proceed to pass the invalid story_chunk to the grapher ingestion queue but should instead return the error to the recording service client

Done. Issue #166 added.